packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -199,7 +199,9 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs,
     }
 
     // Optimize the query operation
-    operation = (await this.mediatorOptimizeQueryOperation.mediate({ context, operation })).operation;
+    const mediatorResult = await this.mediatorOptimizeQueryOperation.mediate({ context, operation });
+    operation = mediatorResult.operation;
+    context = mediatorResult.context || context;
```

Comment:
I don't understand why coverage doesn't flag this line as tested. The test I've added would not work without this.

Reply:
Not sure myself, but perhaps my suggested changes to the test could help finding the cause.

Reply:
Done! Added a dedicated unit tests for the left side of  `||`.